src/Microsoft.DotNet.Build.Tasks/PackageFiles/resources.targets

```diff
@@ -26,14 +26,18 @@
           Condition="'$(StringResourcesPath)'!='' AND '$(OmitResources)'!='true'"
           Inputs="$(StringResourcesPath)"
           Outputs="$(IntermediateResOutputFileFullPath)">
-    
-    
 
     <GenerateResourcesCode
         ResxFilePath="$(StringResourcesPath)" 
         OutputSourceFilePath="$(IntermediateResOutputFileFullPath)" 
         AssemblyName="$(AssemblyName)" />
 
+    <Copy Condition="'$(BuildingUAPVertical)' == 'true'"
+          SourceFiles="$(StringResourcesPath)"
+          DestinationFiles="$(ResourcesFolderPath)\$(AssemblyName).resw"
```

Comment:
I just tried this locally with makepri and used the produced resources.pri inside the runner and ran a library that loads a string from ResourceManager and it works perfectly.